### DISCUSSION TITLE
How can I make contact for only one person I need?

### DISCUSSION THREAD

**Discussion by FarJutharat** [stasel/WebRTC-iOS#83] [Q&A]
I have the back-end to connect WebRTC , I am creating WebRTC client and I have user_id It is an indication of who to call. 
I don't know how implement code to target the call. please suggest me 
You can contact me through farjutuy@gmail.com
I will send my project for you.
thank a lot.

**Comment by stasel** [comment]
Hi FarJutharat, as much as I would like to help, I am not providing that kind of support for this project. If you have any specific question you can try to ask it here and I'll do my best to answer you

**Comment by FarJutharat** [comment]
@stasel  Now , I am create WebRTC with SwiftUI. I am having trouble making a video camera launcher. How can I implement startCaptureLocalVideo() function with swiftUI? I am trying to follow this issue  https://stackoverflow.com/questions/60087973/webrtc-videocall-rendering-in-swiftui/60109579#60109579 . But I don't understand how it can be used. If you can show an example of how to create a structure to display RTCVideo on swiftUI, I would appreciate it very much.
